File: Lista 1/Multimetro.java
import java.util.Scanner;
import java.lang.Math;

public class Multimetro{
    // Numero de amostras
    static final int NUM_AMOSTRAS = 10;

    public static void main(String[] args){
        try (Scanner scanner = new Scanner(System.in)){
            // Entrada de dados por teclado
            // double[] volts = new double[NUM_AMOSTRAS];
            // for(int i=0; i<NUM_AMOSTRAS; i++){
            //     System.out.print("Amostra " + (i+1) + ": ");
            //     volts[i] = scanner.nextDouble();
            // }
            double[] volts = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};

            // Amostra de dados
            System.out.print("\nAmostra de Volts: ");
            for(double num : volts)
                System.out.print(num + ", ");

            // Cálculo da media
            double media = calcularMedia(volts, NUM_AMOSTRAS);
            System.out.println("\nMedia: " + media);
            
            // Cálculo do desvio padrão
            double desvioPadrao = calcularDesvioPadrao(volts, media, NUM_AMOSTRAS);
            System.out.println("Desvio Padrão: " + desvioPadrao);
            
            // Situação do multímetro
            System.out.println("Situação: " + ((desvioPadrao > media*0.1) ? "Multímetro com problema." : "Multímetro OK."));
        }
    }

    public static double calcularMedia(double[] amostras, int numAmostras){
        double soma = 0;
        for(double amostra : amostras)
            soma += amostra;
        return soma/numAmostras;
    }

    public static double calcularDesvioPadrao(double[] amostras, double media, int numAmostras){
        double soma = 0;
        for(double amostra : amostras)
            soma += Math.pow(amostra - media, 2);
        return Math.sqrt(soma/numAmostras);
    }
}   

File: Lista 2/Multimetro.py
def calcularMedia(lista):
    return sum(lista)/len(lista)
    
def calcularDesvioPadrao(lista, media):
    from math import sqrt
    return sqrt(sum((numero - media)**2 for numero in lista)/len(lista))
    
def main():
    NUM_AMOSTRAS = 5
    
    # Entrada de dados por teclado
    # amostras = []
    # i = 0
    # while i < NUM_AMOSTRAS:
    #     try:
    #         amostras.append(float(input(f'Amostra {i+1}: ')))
    #         i += 1
    #     except:
    #         print('Valor inválido. Por favor, insira um número.')
    amostras = [1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0]
            
    print('\nAmostras: ', end=' '); print(*amostras, sep=', ')

    # Cálculo da média
    media = calcularMedia(amostras)
    print('Media: ', media)

    # Cálculo do desvio padrão
    desvioPadrao = calcularDesvioPadrao(amostras, media)
    print('Desvio Padrão: ', desvioPadrao)

    # Situação do multímetro
    print('Situação: ', 'Multímetro com problema.' if desvioPadrao > media*0.1 else 'Multímetro OK.')


if __name__ == '__main__':
    main()